lib/puppetserver/ca/action/enable.rb

```diff
@@ -10,25 +10,27 @@
 module Puppetserver
   module Ca
     module Action
-      class InfraCRL
+      class Enable
         include Puppetserver::Ca::Utils
 
         SUMMARY = "Setup infrastructure CRL based on a node inventory."
         BANNER = <<-BANNER
 Usage:
-  puppetserver ca infracrl [--help]
-  puppetserver ca infracrl [--config PATH]
+  puppetserver ca enable [--help]
+  puppetserver ca enable [--config PATH] [--infracrl]
 
 Description:
-  Creates auxiliary files needed to use the infrastructure-only CRL.
-  Assumes the existence of an `infra_inventory.txt` file in the CA
-  directory listing the certnames of the infrastructure nodes in the
-  Puppet installation. Generates the `infra_serials` file and the empty
-  CRL to be populated with revoked infrastructure nodes.
+  Performs actions necessary to enable certain CA modes.
 
-  To determine the target location, the default puppet.conf
-  is consulted for custom values. If using a custom puppet.conf
-  provide it with the --config flag
+  The default puppet.conf is queried for configuration If using a custom
```

Comment:
This is a flag that everything has but this is the only place we document it like this. I'd remove this section and if you think we need to describe it more see if this works in the general help output.

Reply:
Decided to just drop it, I think the option help text is sufficient.